crates/re_viewport/src/viewport.rs

```diff
@@ -262,6 +263,96 @@ impl<'a, 'b> egui_tiles::Behavior<SpaceViewId> for TabViewer<'a, 'b> {
         text
     }
 
+    #[allow(clippy::fn_params_excessive_bools)]
+    fn tab_ui(
+        &mut self,
+        tiles: &egui_tiles::Tiles<SpaceViewId>,
+        ui: &mut egui::Ui,
+        id: egui::Id,
+        tile_id: egui_tiles::TileId,
+        active: bool,
+        is_being_dragged: bool,
+    ) -> egui::Response {
+        // custom tab UI when we have a space view
+        let Some(egui_tiles::Tile::Pane(space_view_id)) = tiles.get(tile_id) else {
+            return egui_tiles::Behavior::<SpaceViewId>::tab_ui(
+                self,
+                tiles,
+                ui,
+                id,
+                tile_id,
+                active,
+                is_being_dragged,
+            );
+        };
+
+        // tab icon
+        let space_view = self
+            .space_views
+            .get(space_view_id)
+            .expect("Should have been populated beforehand");
+        let icon = space_view.class(self.ctx.space_view_class_registry).icon();
+        let image = self.ctx.re_ui.icon_image(icon);
+        let texture_id = image.texture_id(ui.ctx());
+        let text_to_icon_padding = 4.0;
```

Comment:
didn't we want to make this a constant on re_ui? guess that's a follow-up

Reply:
Right. This can become a constant. It'd be an improvement even against the status quo.